scripts/validate_docstrings.py

```diff
@@ -181,18 +184,24 @@ def doc_parameters(self):
 
     @property
     def signature_parameters(self):
-        if (inspect.isclass(self.method_obj)
-                and self.method_name.split('.')[-1] in
-                self.method_obj._accessors):
-            # accessor classes have a signature, but don't want to show this
-            return tuple()
+        if inspect.isclass(self.method_obj):
+            if hasattr(self.method_obj, '_accessors') and (
+                    self.method_name.split('.')[-1] in
+                    self.method_obj._accessors):
+                # accessor classes have a signature but don't want to show this
```

Comment:
This does not seem to work for me. Eg checking `pandas.Series.dt` with the script complains about parameter 'data' not being documented.

(but anyhow, it is better as on master, there it raises an attribute error somewhere in the script)

Reply:
Good to know. I’d be happy to open that up as a follow up issue here - worth adding dedicated tests for accessors